pkg/master/master.go

```diff
@@ -216,6 +216,10 @@ type Config struct {
 	// Defaults to 6443 if not set.
 	ReadWritePort int
 
+	// The name of the port on PublicAddress and master service port
+	// Defaults to "https" if not set.
+	ReadWritePortName string
```

Comment:
No, port 443 will always be there so we could just assume it's going to be
named "https".

On Wednesday, October 7, 2015, Tim Hockin <notifications@github.com> wrote:

> In pkg/master/master.go
> <https://github.com/kubernetes/kubernetes/pull/14871#discussion_r41410569>
> :
>
> > @@ -216,6 +216,10 @@ type Config struct {
> >  	// Defaults to 6443 if not set.
> >  	ReadWritePort int
> >
> > +	// The name of the port on PublicAddress and master service port
> > +	// Defaults to "https" if not set.
> > +	ReadWritePortName string
>
> Do we really need to parameterize this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14871/files#r41410569>.
>